org.freedesktop.Sdk.Extension.texlive.appdata.xml

```diff
@@ -6,6 +6,7 @@
   <summary>LaTeX packages and tools</summary>
   <url type="homepage">https://tug.org/texlive/</url>
   <releases>
-    <release date="2020-04-06" version="2020.04.06" />
+    <release version="20210325" date="2021-03-25"/>
```

Comment:
why removing the dots? Is it because f-e-d-c will mess with them ?

Reply:
This seems to be the release date, not the texlive version which is 2020 according to `version.ac` in `texlive-source.tar.xz`, so if we want to be correct then it should be set to 2020.  

Here are some example from distros:

In Feodra, the package version is 20210325.  
In Arch Linux, the package version is 2021-{svn_rev}.  
In Debian, the package version is 2021.20210921.

What's correct?

The distributed tarballs don't have dots so let's follow that.  
Also, this way it will be in the same format/scheme as the version value generated by f-e-d-c.